node/libs/roles/src/validator/conv.rs

```diff
@@ -57,14 +57,20 @@ impl ProtoFmt for Genesis {
                         .context("validators")?,
                     GenesisVersion(0),
                 )
-            // empty validator set, Committee:new() will later return an error.
+                // empty validator set, Committee:new() will later return an error.
             } else {
                 (vec![], GenesisVersion::CURRENT)
             };
         Ok(Self {
             fork: read_required(&r.fork).context("fork")?,
             validators: Committee::new(validators.into_iter()).context("validators")?,
             version,
+            leader_selection: r
+                .leader_selection
```

Comment:
please verify that Sticky.key actually belongs to the committee

Reply:
...and when the bft actor is starting as well.

Reply:
Fixed here: https://github.com/matter-labs/era-consensus/pull/96/commits/5bd14b38ff3c7d5289800af4d9bf906712d74d5c